modules/kogito-swf/devmode/runtime/osl/module.yaml

```
@@ -0,0 +1,32 @@
+schema_version: 1
+name: org.kie.kogito.swf.devmode.runtime.osl
+version: "1.29.0"
+description: "OpenShift Serverless Logic devmode with required extensions"
+
+envs:
+  - name: QUARKUS_KOGITO_DEVSERVICES_ENABLED
+    description: Whether to enable Kogito Devservices or not. Default to "false" since Data Index needs Docker.
+    value: "false"
+  - name: QUARKUS_DEVSERVICES_ENABLED
+    description: Whether to enable Quarkus Devservices or not. Default to "false" since the Kafka broker needs Docker.
+    value: "false"
+  - name: QUARKUS_KUBERNETES_CLIENT_DEVSERVICES_ENABLED
+    description: Whether to enable Kogito Devservices or not. Default to "false" since Kubernetes objects are managed by the Kogito Serverless Operator.
+    value: "false"
+  - name: QUARKUS_CONTINUOUS_TESTING
+    description: Whether to enable Quarkus continous testing on DevMode or not. Default to "disabled".
+    value: "disabled"
+  - name: KOGITO_CODEGEN_PROCESS_FAILONERROR
+    description: Boolean flag that will prevent Dev Mode to crash if the provided workflow is wrong.
+    value: "false"
+
+artifacts:
+  - path: /home/kogito/build_output/kogito-swf-quarkus-app.zip
+    # kogito-devmode-quarkus-app-1.38.0.Final-redhat-00001-image-build.zip
+    md5: c67fa5b0c78688023a5ac39877415dc7
+  - path: /home/kogito/build_output/kogito-swf-maven-repo.zip
+    # kogito-devmode-maven-repository-1.38.0.Final-redhat-00001-image-build.zip
+    md5: 5452062f247963d2ef7dbc35fbe29c3b
```

Just a suggestion, can we use `name` instead of `path` for the artifacts? Same way done for the data-index image (see [kogito-data-index-ephemeral module.yaml](https://github.com/kiegroup/kogito-images/blob/main/modules/kogito-data-index-ephemeral/prod/module.yaml#L5))

From prod side I guess it is clear,  specially when we integrate this build into CPaaS.


```suggestion
  # kogito-devmode-quarkus-app-1.38.0.Final-redhat-00001-image-build.zip
  - name: kogito-devmode-quarkus-app-image-build.zip
    md5: c67fa5b0c78688023a5ac39877415dc7
  # kogito-devmode-maven-repository-1.38.0.Final-redhat-00001-image-build.zip
  - name: kogito-devmode-maven-repository-image-build.zip
    md5: 5452062f247963d2ef7dbc35fbe29c3b
```